### PR TITLE
[HPRO-1041] Moved footer container/renamed template

### PR DESCRIPTION
**Pull request by Ehaic**
| Q                   | A
| ------------------- | --------------
| Target Release?     | Next available 
| Bug fix?            | ❌ 
| New feature?        | ❌ 
| Database migration? | ❌ 
| New configuration?  | ❌ 
| Composer updates?   | ❌ 
| NPM updates?        | ❌ 
| Jira Ticket(s)      | HPRO-1041 

### Summary

Definitely looking for feedback on this one. Wasn't sure the best way to do this. I wanted to keep using the help twig that already exists so I moved the footer container outside of the twig and on the help page where it needs moved to the top put it inside a new div.

Also wasn't sure how to label this.

### CODE REVIEW

File: templates/help/faq.html.twig [hunk 1]
```diff
@@ -10,6 +10,8 @@
         <h2><i class="fa fa-question-circle-o" aria-hidden="true"></i> Technical FAQs</h2>
     </div>
     <a href="{{ faqs.web_link }}" target="_blank" class="btn btn-primary"><i class="fa fa-external-link" aria-hidden="true"></i> Click here to visit the HealthPro FAQ document</a>
-    {% include 'help/footer.html.twig' %}
+<footer class="footer help-footer">
```
Comment (thornview) [inline templates/help/faq.html.twig:13]: I would suggest keeping the `footer.html.twig` file for all the places that still need the footer, but extracting the `helptext.html.twig`, as you've done. The footer template would load the helptext template, so it's a little bit nested, but avoids duplication.

**Review by thornview — CHANGES_REQUESTED** [review]

File: templates/help/faq.html.twig (hunk 1 above)
Comment (Ehaic) [reply templates/help/faq.html.twig:13]: Awesome Thanks for the feedback, I'll make those changes.

File: templates/help/footer.html.twig [hunk 2]
```diff
@@ -1,6 +1,3 @@
-<footer class="footer help-footer">
-    <div align="center">
-        <p class="text-muted">For further help with HealthPro, please contact 844-842-2854. The Support Center will be accessible from Monday - Friday, 7am - 10pm EST.</p>
-        {% include 'partials/drc-support-text.html.twig' %}
-    </div>
-</footer>
+<footer class="footer help-footer">
+    {%include 'help/helptext.html.twig'%}
+</footer>
```
Comment (thornview) [inline templates/help/footer.html.twig:3]: GitHub fusses if you don't have a blank line at the end of your files. 
  
```suggestion
</footer>

```

File: templates/help/helptext.html.twig [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+<div align="center">
+    <p class="text-muted">For further help with HealthPro, please contact 844-842-2854. The Support Center will be accessible from Monday - Friday, 7am - 10pm EST.</p>
+    {% include 'partials/drc-support-text.html.twig' %}
+</div>
```
Comment (thornview) [inline templates/help/helptext.html.twig:5]: Another blank line missing. 

```suggestion
</div>

```

**Review by thornview — CHANGES_REQUESTED** [review]
Need to add blank lines to the end of the files then this looks good.

File: templates/help/helptext.html.twig [hunk 4]
```diff
@@ -0,0 +1,5 @@
+
```
Comment (shyamzcode) [inline templates/help/helptext.html.twig:1]: We could remove this extra line at the top?

**Pull request closed without merging**